src/material/button/button.md

```diff
@@ -12,26 +12,15 @@ There are several button variants, each applied as an attribute:
 
 | Attribute            | Description                                                              |
 |----------------------|--------------------------------------------------------------------------|
-| `mat-button`         | Rectangular text button w/ no elevation                                  |
-| `mat-raised-button`  | Rectangular contained button w/ elevation                                |
-| `mat-flat-button`    | Rectangular contained button w/ no elevation                             |
-| `mat-stroked-button` | Rectangular outlined button w/ no elevation                              |
+| `mat-button`         | Rectangular text button w/ no elevation and rounded corners                                 |
```

Comment:
Not sure if we should mention the appearance here since it can change based on the tokens. I know this was here before so totally fine to also just leave it.

Reply:
Yeah I was on the fence about that too, but technically the M3 specs call out that it has rounded corners and that is the default